src/protobuf/java/com/datadoghq/sketch/ddsketch/store/StoreProtoBinding.java

```diff
@@ -0,0 +1,60 @@
+package com.datadoghq.sketch.ddsketch.store;
+
+import java.util.function.Supplier;
+
+public class StoreProtoBinding {
+
+    /**
+     * Generates a protobuf representation of this {@code Store}.
+     *
+     * @return a protobuf representation of this {@code Store}
+     */
+    public static com.datadoghq.sketch.ddsketch.proto.Store toProto(Store store) {
+        if (store instanceof DenseStore) {
+            return toProtoDense((DenseStore) store);
+        }
+        return toProtoSparse(store);
+    }
+
+    private static com.datadoghq.sketch.ddsketch.proto.Store toProtoSparse(Store store) {
+        final com.datadoghq.sketch.ddsketch.proto.Store.Builder storeBuilder =
+                com.datadoghq.sketch.ddsketch.proto.Store.newBuilder();
+        // In the general case, we use the sparse representation to encode bin counts.
+        store.getStream().forEach(bin -> storeBuilder.putBinCounts(bin.getIndex(), bin.getCount()));
```

Comment:
Can't we now use `forEach`?
```suggestion
        store.forEach((index, count) -> storeBuilder.putBinCounts(index, count));
```